src/machine/asm/execute_aarch64.S

```diff
@@ -1594,6 +1611,66 @@ ckb_vm_x64_execute:
   WRITE_RS1(TEMP1)
   WRITE_RD(TEMP3)
   NEXT_INST
+.CKB_VM_ASM_LABEL_OP_ADCS:
+  DECODE_R4
+  mov TEMP1, 0
+  ldr TEMP3, REGISTER_ADDRESS(RS1)
+  ldr TEMP4, REGISTER_ADDRESS(RS2)
+  adds TEMP3, TEMP3, TEMP4
+  adc TEMP1, TEMP1, TEMP1
+  WRITE_RD(TEMP3)
+  WRITE_RS3(TEMP1)
+  NEXT_INST
+.CKB_VM_ASM_LABEL_OP_SBBS:
+  DECODE_R4
+  mov TEMP1, 0
+  ldr TEMP3, REGISTER_ADDRESS(RS1)
+  ldr TEMP4, REGISTER_ADDRESS(RS2)
+  subs TEMP3, TEMP3, TEMP4
+  adc TEMP1, TEMP1, TEMP1
+  eor TEMP1, TEMP1, 1
```

Comment:
Why xored TEMP1?

Reply:
arm64 uses different borrow logic in subtraction here: when a borrow happened, the carry bit is 0, otherwise the carry bit is 1. This is the opposite of what we want, hence the xor.

Reply:
Understand, it is said to simplify chip design

https://stackoverflow.com/questions/53065579/confusion-about-arm-documentation-on-carry-flag